bin/start-kuzzle-server

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+
+/*
+ * Kuzzle, a backend software, self-hostable and ready to use
+ * to power modern apps
+ *
+ * Copyright 2015-2018 Kuzzle
+ * mailto: support AT kuzzle.io
+ * website: http://kuzzle.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const
+  fs = require('fs'),
+  Kuzzle = require('../lib/api/kuzzle'),
+  ColorOutput = require('./colorOutput'),
+  yargs = require('yargs');
+
+function loadJson (filePath) {
+  const rawData = fs.readFileSync(filePath, 'utf8');
+
+  return JSON.parse(rawData);
+}
+
+function startKuzzle (options = {}) {
+  const
+    kuzzle = new Kuzzle(),
+    cout = new ColorOutput(options),
+    kuzzleParams = {};
+
+  cout.notice('[ℹ] Starting Kuzzle server');
+
+  if (options.mappings) {
+    kuzzleParams.mappings = loadJson(options.mappings);
+  }
+
+  if (options.fixtures) {
+    kuzzleParams.fixtures = loadJson(options.fixtures);
+  }
+
+  if (options.securities) {
+    kuzzleParams.securities = loadJson(options.securities);
+  }
+
+  if (options.enablePlugins) {
```

Comment:
This block is outdated. (see: https://github.com/kuzzleio/kuzzle/blob/2-dev/bin/commands/start.js)